### DISCUSSION TITLE
Suggestion to add a manually refresh button

### DISCUSSION THREAD

**Discussion by FindHao** [iterate-ch/cyberduck#14853] [Ideas]
Hi, thanks for your work! It is an amazing tool to help me check the files via sftp. However, I found sometimes the file list has some delays as much as 10s seconds. Is it possible to add a button for manually refreshing? Thanks.

**Comment by dkocher** [comment]
There is a _Refresh_ button in the toolbar of the Cyberduck [browser window](https://docs.cyberduck.io/cyberduck/browser/) and a _Reload_ option in the Mountain Duck [context menu](https://docs.cyberduck.io/mountainduck/interface/#reload).

**Reply by FindHao** [reply]
Thanks! I found the `Reload` option in the Mountain Duck!
